glusterd2/store/remote.go

```diff
@@ -1,19 +1,50 @@
 package store
 
 import (
+	"crypto/tls"
+	"crypto/x509"
+	"io/ioutil"
 	"time"
 
 	"github.com/coreos/etcd/clientv3"
 	log "github.com/sirupsen/logrus"
 )
 
 func newRemoteStore(conf *Config) (*GDStore, error) {
-
-	c, e := clientv3.New(clientv3.Config{
+	var tlsConfig *tls.Config
+	var c *clientv3.Client
+	var e error
+	if conf.UseTLS {
+		tlsConfig = &tls.Config{
+			MinVersion: tls.VersionTLS10,
```

Comment:
These are the defaults for these. There is no need to set them unless you change to something different. I'd prefer if we just stick to tls1.2, but that's just my opinion.

Reply:
Ack

Reply:
@kshlm do you want min version to be set as 1.2?